components/grid/editing/popup.md

```diff
@@ -210,6 +210,8 @@ The `GridPopupEditFormSettings` nested tag exposes the following parameters to a
 
 @[template](/_contentTemplates/common/popup-edit-customization.md#edit-form-settings)
 
+>important These settings are not applicable if you are using a `<FormTemplate>` with a custom Form component. See more details in [FormTemplate - Specifics]({%slug grid-templates-popup-form%}#specifics).
```

Comment:
Don't use the PascalCase form when you are not referring to the template name as used in the code. So, in titles, use "Form Template".
```suggestion
>important These settings are not applicable if you are using a `<FormTemplate>` with a custom Form component. See more details in [Form Template - Specifics]({%slug grid-templates-popup-form%}#specifics).
```